dist/tools/riot_hdr/common.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2017 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU General Public
+ * License v2. See the file LICENSE for more details.
+ */
+
+#ifndef COMMON_H
+#define COMMON_H
+
+#include <unistd.h>
+#include <sys/types.h>
```

Comment:
For `size_t` including `stddef.h` is enough.

Reply:
Will change.